ansible/roles/passwords/tasks/main.yml

```diff
@@ -7,6 +7,19 @@
   delegate_to: localhost
   run_once: true
 
+- name: Generate k3s token
+  ansible.builtin.set_fact:
+    k3s_token_secret: ""
+
+- name:  Generate k3s token and add to terraform
+  vars:
+    token: "{{ lookup('ansible.builtin.password', '/dev/null', length=64) }}"
```

Comment:
Nope. Lets create this in the same place as the other secrets, ansible/roles/passwords/defaults/main.yml